src/robot/run.py

```diff
@@ -464,6 +464,9 @@ def run(*datasources, **options):
     passed as lists like `include=['tag1', 'tag2']`. If such option is used
     only once, it can be given also as a single string like `include='tag'`.
 
+    Additionally listener option allows to pass object directly instead of
+    module name, e.g. `run('tests.robot', listener=Listener())`.
```

Comment:
Listeners aren't typically implemented as a module. It would be better to change `module name` to `listener name`. Also `allows to pass` doesn't sound that good. Perhaps `allows passing` would be better.